src/main/java/school/lemon/changerequest/java/generics/NumberGenerator/Printer.java

```diff
@@ -0,0 +1,31 @@
+package school.lemon.changerequest.java.generics.NumberGenerator;
+
+import java.util.Arrays;
+
+/**
+ * Created by lera on 27.12.16.
+ */
+public class Printer<T extends Number> {
+
+    NumberGeneratorImp<T> numb;
+
+    public Printer(NumberGeneratorImp numb1) {
+        this.numb = numb1;
+    }
+
+    public T generateAndPrint() {
```

Comment:
Looks like this method doesn't print anything. So there are 2 ways: rename method or move `System.out.println` inside.